apps/kimi/CLAUDE.md

```
@@ -0,0 +1,163 @@
+
+Mapping to our internal format:
+
+| Kimi Field             | Internal Field          | Calculation                                   |
+| ---------------------- | ----------------------- | --------------------------------------------- |
+| `input_other`          | `inputTokens`           | Part of total input                           |
+| `input_cache_read`     | `cachedInputTokens`     | Also added to `inputTokens`                   |
+| `input_cache_creation` | (part of input)         | Added to `inputTokens`                        |
+| `output`               | `outputTokens`          | Direct mapping                                |
+| `ContentPart.think`    | `reasoningOutputTokens` | Estimated from think content (~4 chars/token) |
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)